package.json

```diff
@@ -14,7 +14,8 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "lint": "eslint src/**/*.js"
   },
```

Comment:
Add a jest key after scripts to ignore certain files when running the test  
E.g: `"jest": {
		"coveragePathIgnorePatterns": [
			"/src/serviceWorker.js",
			"/src/index.jsx"
		]
	}`